cli/cmd/user/remove.go

```diff
@@ -0,0 +1,64 @@
+package user
+
+import (
+	"fmt"
+
+	"github.com/rilldata/rill/cli/cmd/cmdutil"
+	"github.com/rilldata/rill/cli/pkg/config"
+	adminv1 "github.com/rilldata/rill/proto/gen/rill/admin/v1"
+	"github.com/spf13/cobra"
+)
+
+func RemoveCmd(cfg *config.Config) *cobra.Command {
+	var orgName string
+	var projectName string
+	var email string
+
+	removeCmd := &cobra.Command{
+		Use:   "remove",
+		Short: "Remove",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if orgName == "" {
+				orgName = cfg.Org
+			}
+
+			client, err := cmdutil.Client(cfg)
+			if err != nil {
+				return err
+			}
+			defer client.Close()
+
+			cmdutil.StringPromptIfEmpty(&email, "Please enter the email of the user.")
```

Comment:
1. Move to before creation of client (may save a bit of latency, not sure if it does a handshake on open).
2. Shorten to `Enter email`